tools/travis/build.sh

```diff
@@ -6,6 +6,7 @@ set -e
 SCRIPTDIR=$(cd $(dirname "$0") && pwd)
 ROOTDIR="$SCRIPTDIR/../.."
 HOMEDIR="$SCRIPTDIR/../../../"
+export PYTHONPATH=/opt/python/2.7.13/lib/python2.7/site-packages
```

Comment:
there are two versions of python installed per the travis logs (3.x and 2.7) this forces the latter to be used.